this creates a /var/tmp/vi.recover on the *host* during packaging,
which does not actually do anything for the package itself, but
does create a useless directory on the host itself and fail when
var/tmp is readonly.
--
diff --git a/GNUmakefile b/GNUmakefile
index 97f34e5..022565e 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -584,13 +584,6 @@ install: bin/vi bin/ex bin/view docs/USD.doc/vi.man/vi.1  \
 ifndef DEBUG
 	-@$(PRINTF) "\r\t%s\t%42s\n" "mkdir:" "/var/tmp/vi.recover"
 endif # DEBUG
-	@$(VERBOSE); $(TEST) -d "/var/tmp/vi.recover" ||  \
-          {                                               \
-            $(MKDIR) "/var/tmp/vi.recover" &&             \
-            $(TEST) -d "/var/tmp/vi.recover" &&           \
-            $(CHOWN) "$(IUSGR)" "/var/tmp/vi.recover" &&  \
-            $(CHMOD) "1777" "/var/tmp/vi.recover";        \
-          }
 ifndef DEBUG
 	-@$(PRINTF) "\r\t%s\t%42s\n" "mkdir:" "$(PREFIX)/bin"
 endif # DEBUG
